### PR TITLE
Update translation for a New Native/Web Сlient

### PR DESCRIPTION
**Pull request by Vafin84**
Please accept my transfer to the main branch of development.

### CODE REVIEW

File: shared/i18n/info.json [hunk 1]
```diff
@@ -61,7 +61,7 @@
       "country_code": "ru",
       "path": "ru_translate_vafin.translation",
 
-      "name": "Russion translation by Vafin, baste on google translate",
+      "name": "Русский (Перевод от 1GTS.RU)",
```
Comment (WolverinDEV) [inline shared/i18n/info.json]: Please remove the self advertisement :)
I will may add something like extra notes or description for that, but within in title its inappropriate)

File: shared/i18n/ru_translate_vafin.translation [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "info": {
-    "name": "Русский язык / Russian language",
+    "name": "Русский (Перевод от 1GTS.RU)",
```
Comment (WolverinDEV) [inline shared/i18n/ru_translate_vafin.translation]: Same as above.

File: shared/i18n/info.json (hunk 1 above)
Comment (Vafin84) [reply shared/i18n/info.json]: > Please remove the self advertisement :)
> I will may add something like extra notes or description for that, but within in title its inappropriate)

How do I change the data already in the open PR? Or need to create a new PR?

Comment (Vafin84) [reply shared/i18n/info.json]: I removed the link to the site

**Pull request merged**